### ISSUE TITLE
Manually deleting installation dir confuses kerl

### ISSUE THREAD

**Issue by RasmusWL** [kerl/kerl#236]
I had to rebuild a version of erlang that was already installed using kerl. For some reason I deleted the folder containing the installation myself. After having build the new version, I was unable to install in the same location, as shown here:

Unable to install
```
$ kerl install r17.5 ~/kerl/r17.5
ERROR: Installation (r17.5) already registered for this location (/home/rasmuswl/kerl/r17.5)
```

Unable to delete
```
$ kerl delete installation ~/kerl/r17.5
/home/rasmuswl/kerl/r17.5 is not a kerl-managed Erlang/OTP installation
```

---

This is definitely not a high priority thing to fix, just wanted to let you know :)

ps. thanks for the good work :heart:

**Comment by nalundgaard** [comment]
For what it's worth, this is just maintained in ~/.kerl/otp_installations file. It's just a 2-column file with build names and installation paths. If you nuked an installation, you can make kerl forget about it by editing this file and be on your way.

That seems OK to me, since IMO expecting the library to tolerate nondeterministic external manipulation of its installations sounds like a slippery slope. I'd prefer to just beef up the [how it works](https://github.com/kerl/kerl#how-it-works), or maybe add a Troubleshooting section rather than make the script handle this kind of thing

**Comment by mrallen1** [comment]
Thanks for reporting the issue. I agree in part and disagree in part with @nalundgaard - I think kerl ought to take care of the simplest case automatically - that is, the directory in the install file doesn't exist any more. It should remove the entry from its own files and do nothing else.

I tend to think any other scenario is not very safe for automation and probably should have some additional documentation in the README.

**Comment by mrallen1** [comment]
Believe this may be a dup of #216

**Comment by arcseldon** [comment]
Good to have an easy workaround, but agree ideally this would be automated.

**Comment by Stratus3D** [comment]
We encounter this with asdf-erlang too https://github.com/asdf-vm/asdf-erlang/issues/143

**Comment by paulo-ferraz-oliveira** [comment]
As @jadeallenx hints at, this is a duplicate of #216, so will be solved by the same pull request: #490.
